server/data/rpClient.ts

```diff
@@ -2,12 +2,18 @@ import RestClient from './restClient'
 import config from '../config'
 
 export default class RPClient {
-  private static restClient(token: string): RestClient {
-    return new RestClient('RP API Client', config.apis.rpClient, token)
```

Comment:
Just wondering if this change is going to have any performance impact? Before we were creating a new RestClient object for each call but now it's the same one re-used. This pattern was taken from the dps bootstrap so wondering if it was done this way for a reason?